standalone/src/models/custom-fields/custom-fields/custom-fields-draft/builders.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { TBuilder } from '@/core';
+import { TCtpCustomFieldInput } from '@/graphql-types';
+import { ReferenceDraftRest, ReferenceDraftGraphql } from '@/models/commons';
+import type {
+  TCustomFieldsDraftRest,
+  TCustomFieldsDraftGraphql,
+} from '../types';
+import { RestModelBuilder, GraphqlModelBuilder } from './builders';
+
+const populateRestModel = (model: TBuilder<TCustomFieldsDraftRest>) =>
+  model.type(ReferenceDraftRest.presets.typeReference()).fields({
+    stringField: 'draft test value',
+    booleanField: false,
+    numberField: 123,
+  });
+
+const populateGraphqlModel = (model: TBuilder<TCustomFieldsDraftGraphql>) =>
+  model
+    .type(ReferenceDraftGraphql.presets.typeReference())
+    .typeId('test-type-id')
+    .typeKey('test-type-key')
+    .fields([
+      { name: 'stringField', value: JSON.stringify('draft test value') },
+      { name: 'booleanField', value: JSON.stringify(false) },
+      { name: 'numberField', value: JSON.stringify(123) },
```

Comment:
I was expecting consumers to use this syntax:
```
      { name: 'stringField', value: 'draft test value' },
      { name: 'booleanField', value: 'false' },
      { name: 'numberField', value: '123' },
```

Perhaps I'm missing something but why do we need to use `JSON.stringify`?